lib/main.dart

```diff
@@ -1,11 +1,14 @@
 import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/foundation.dart' show kDebugMode;
 import 'package:flutter/material.dart';
 import 'package:movies_and_stars/di/app_injector.dart';
 import 'package:presentation/star_movie_app.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp();
+  await FirebaseCrashlytics.instance.setCrashlyticsCollectionEnabled(kDebugMode);
```

Comment:
```suggestion
  await FirebaseCrashlytics.instance.setCrashlyticsCollectionEnabled(!kDebugMode);
```

u should have crashlytics only for release builds

Reply:
resolved